src/js/App.vue

```diff
@@ -41,6 +41,7 @@ import { Content } from '@nextcloud/vue'
 import { subscribe, unsubscribe } from '@nextcloud/event-bus'
 import { mapState } from 'vuex'
 import '@nextcloud/dialogs/styles/toast.scss'
+import './assets/scss/hacks.scss'
```

Comment:
:see_no_evil: 
Sorry you had to do that, I'm reverting :p

Reply:
shame on you. 😠